helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/client/HelpDeskStreamsClient.java

```diff
@@ -31,15 +31,15 @@ public HelpDeskStreamsClient(SymAuth symAuth, SymphonyClientConfig config) {
     this.apiClient.setBasePath(config.get(SymphonyClientConfigID.POD_URL));
   }
 
-  public SymRoomDetail createChatRoom(SymRoomAttributes roomAttributes, boolean roomHistory) throws StreamsException {
+  public SymRoomDetail createChatRoom(SymRoomAttributes roomAttributes) throws StreamsException {
```

Comment:
Please check if this method still required or you can use the method from the superclass.